src/lib/sci/README.md

```diff
@@ -7,11 +7,15 @@ It represents the amount of carbon emitted per
 
 ## Parameters
 
-### Model config
+### Plugin global config
 
 - `functional-unit`: the functional unit in which to express the carbon impact
 - `functional-unit-time`: the time to be used for functional unit conversions, as a string composed of a value and a unit separated with a space, hyphen or underscore, e.g. `2 mins`, `5-days`, `3_years`
 
+### Plugin node config
```

Comment:
does this override the `functional-unit` in global config? If this is the behaviour, then it should also apply to `functional-unit-time`

Reply:
yes, it's override. Asim declared in the spreadsheet that `functional-unit` should be either global-config or node/config, `functional-unit-time` should be in node config

Reply:
can you clarify that in the text, please

Reply:
added additional description